src/CachePlugin.php

```diff
@@ -138,6 +137,34 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * @param int|null $maxAge
+     *
+     * @return int|null
+     */
+    private function calculateCacheItemExpiresAfter($maxAge)
+    {
+        if ($this->config['cache_lifetime'] === null && $maxAge === null) {
+            return;
+        }
+
+        return $this->config['cache_lifetime'] + $maxAge;
+    }
+
+    /**
```

Comment:
lets add phpdoc here. "Calculate the timestamp when a response expires. After that timestamp, we need to send a If-Modified-Since / If-None-Match request to validate the response"